packages/plugins/typescript/document-nodes/src/index.ts

```diff
@@ -0,0 +1,122 @@
+import { PluginFunction, PluginValidateFn, Types } from '@graphql-codegen/plugin-helpers';
+import { GraphQLSchema, OperationDefinitionNode } from 'graphql';
+import { print } from 'graphql/language/printer';
+
+export type NamingConvention = 'lowerCamelCase' | 'UpperCamelCase' | 'UPPER_CASE';
+
+function upperFirst(str: string): string {
+  return str[0].toUpperCase() + str.slice(1);
+}
+
+function camelize(str: string): string {
```

Comment:
You can use `change-case` library for those transformation, it's already part of the codegen's dependencies.

Reply:
+1

Reply:
I'm going to change this parameter to be compatible with namingConvention from visitor-plugin (`module#method`) so it will be more flexible